src/QMCWaveFunctions/AGPDeterminant.cpp

```diff
@@ -248,37 +248,27 @@ AGPDeterminant::evaluateLogAndStore(ParticleSet& P)
   d2Y = GeminalBasis->d2Y;
 }
 
-AGPDeterminant::ValueType
-AGPDeterminant::registerData(ParticleSet& P, PooledData<RealType>& buf)
+void
```

Comment:
Do we have any tests to indicate that AGPDeterminant is functional pre/post modification?

Reply:
We don't have tests. But my change should be low risk see my explanation comment below.